ceph-ansible.spec.in

```diff
@@ -35,6 +35,14 @@ for f in ansible.cfg *.yml *.sample group_vars roles library plugins infrastruct
   cp -a $f %{buildroot}%{_datarootdir}/ceph-ansible
 done
 
+# Strip coreos files.
+# These are unneeded on RPM-based distros, and the playbooks download random
+# things from around the internet.
+pushd %{buildroot}%{_datarootdir}/ceph-ansible
+  rm -r roles/ceph-common-coreos
```

Comment:
Thanks, I see what you mean. RHEL does alias `rm` in root's interactive shell by default.

For RPM builds that is not an issue. Jenkins built the ceph-ansible RPM fine for this branch. https://jenkins.ceph.com/job/ceph-ansible-rpm/631/console , https://shaman.ceph.com/repos/ceph-ansible/rpm-strip-coreos/94be99d158da2861e405f79442537c946e781ba8/default/